graph-migration/README.md

```diff
@@ -5,30 +5,41 @@
 Graph Schema Migration tool creates legacy graph frame. Based on legacy graph frame schema, the tool generates native graph schema.
 
 There are two types of schema, cql and gremlin formats. Cql format schema is the CQL script to create graph keyspace and
-tables. Gremlin format is native graph schema.
-Run the following command to generate native graph schema
+tables. Gremlin format is groovy script to create native graph schema under Gremlin console.
+
+Run the following command to generate native graph schema with DSE server is up:
+```
+dse graph-migrate-schema [-cql | -gremlin] <legacy_graph> <native_graph>
 ```
-dse graph-migrate-schema -cql|-gremlin <legacy_graph> <native_graph>
+
+- `-cql`: specifies the output as cql format.
+- `-gremlin`: specifies the output as gremlin format.
+
+Example:
+```
+dse graph-migrate-schema -gremlin reviewerRating nreviewerRating > native_schema.groovy
 ```
 
-All properties are single. Multi properties are migrated as a single property. Meta properties, Index and MVs are are
+Please note that the following are NOT supported:
+- Meta properties
+- Multi properties
+- Indexes
+
+All properties are single. Multi properties are migrated as a single property. Meta properties, Indexes, and MVs are
 dropped. If custom handling of meta properties and multi properties, renaming properties and dropping properties are
-needed, user should modify the generated native graph schema. The native graph schema can be created through cqlsh or
-gremlin.
+needed, users should modify the generated native graph schema.
```

Comment:
I would reword to something like:

> If custom handling of meta and multi properties is needed, users should modify the generated native graph schema accordingly, some properties may need to be renamed or dropped.

Side note: we really need some examples for dealing with meta and multi properties. Not something we necessarily need in this PR, but something we need to add at some point soon.

Reply:
True, I will leave this comment open.